tests/Integration/Behaviour/Features/Scenario/Order/order_from_bo.feature

```diff
@@ -229,9 +229,9 @@ Feature: Order from Back Office (BO)
     And product "Mug The best is yet to come" in order "bo_order1" has following details:
       | product_quantity            | 3  |
       | product_price               | 12 |
-      | unit_price_tax_incl         | 12 |
+      | unit_price_tax_incl         | 12.72 |
```

Comment:
It's surprising that it's the only place you needed to fix this This indicates that we probably didn't test the OrderDetails values enough

Reply:
well I only had to fix it when there was an "add product" or "edit product" without error before checking the price